cmd/promtool/unittest.go

```diff
@@ -35,15 +36,24 @@ import (
 	"github.com/prometheus/prometheus/promql/parser"
 	"github.com/prometheus/prometheus/rules"
 	"github.com/prometheus/prometheus/storage"
+
+	"github.com/maokt/junitxml"
```

Comment:
Given how simple this is I'd prefer to just add this as another file here for now, makes maintenance easier.

Reply:
this could also be a package in utils/

Reply:
ok, will do that